docs/src/modules/components/overview/Keyboard.tsx

```diff
@@ -410,7 +410,7 @@ export default function Keyboard() {
   const brandingTheme = useTheme();
   const theme = createTheme({ palette: { mode: brandingTheme.palette.mode } });
 
-  const handleKeySelection = (e: React.SyntheticEvent, key: SelectedKey | null) => {
+  const handleKeySelection = (ev: React.SyntheticEvent, key: SelectedKey | null) => {
```

Comment:
Similarly here, I'd for example rename this to "event" and the inner one to "dispatchedEvent"

Reply:
I tend to use `event2` or `eventBis` but your proposal is obviously a looooot better :laughing: